src/python/mshields/chaps/chaps.py

```diff
@@ -113,6 +113,15 @@ def run_goal(args):
   pants_args = "run {0} {1}".format(targets, run_args)
   lib.pants(pants_args)
 
+@app.command(name="clean")
+@app.command(name="clean-all")
+@app.command(name="wash")
+def clean_goal(args):
+  """Clean pants from anywhere in the tree."""
```

Comment:
Chaps mission is to run from anywhere in the tree.  Would prefer to see just: "Clean pants."